src/components/call/Link.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
```

Comment:
`Link` is a `react-router` component. To avoid confusion and add clarity, this component and file name should be `IssueLink`.